### PR TITLE
Updating READMEs

### LINKED ISSUE #114: CounterfactualCovid19 Repo Audit
**NOTE:** Comment in progress as I am working through the install instructions.

Hi All,

I picked this repo to do an audit after I heard James R talking about it earlier today. I attempted to install the software in the ways described in the READMEs. Here are a few things I observed:

* The README mentions a directory for `counterfactual_calculations` that I didn't see on the `develop` branch. I was able to look at the repo that it is supposedly based on, but did not see it in the code base (and I noted that all branches had been merged).
* I tried building the Docker image with `docker-compose`. I got the following error:
  ```
     SSLError

  HTTPSConnectionPool(host='pypi.org', port=443): Max retries exceeded with url: /pypi/astroid/json (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1129)')))

  at /usr/local/lib/python3.9/site-packages/requests/adapters.py:514 in send
      510│                 raise ProxyError(e, request=request)
      511│ 
      512│             if isinstance(e.reason, _SSLError):
      513│                 # This branch is for urllib3 v1.22 and later.
    → 514│                 raise SSLError(e, request=request)
      515│ 
      516│             raise ConnectionError(e, request=request)
      517│ 
      518│         except ClosedPoolError as e:
  ERROR: Service 'backend' failed to build : The command '/bin/sh -c poetry install' returned a non-zero code: 1
  ```

* When installing the frontend dependencies via `npm`, it mentioned some vulnerabilities, some of which were high. This might not b, but it may give someone trying to run the webapp themselves pause. You might note in the readme that this is to be expected?
* I had postgres installed on my system prior to trying to build the backend.

### PR DESCRIPTION
**Pull request by crangelsmith**
Fixes #114

Super minor updates, most of the important changes to the readme were implemented in PR #112

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Once `PostgreSQL` is installed, you can create the required database and the `dj
 
 ```bash
 createdb counterfactualcovid
-psql counterfactualcovid -c "CREATE USER django WITH PASSWORD 'value';" # where you should replace value with the desired password
+psql counterfactualcovid -c "CREATE USER django WITH PASSWORD 'value';"  # where you should replace value with the desired password, the one for this project can be found in line 90 of /backend/settings/common.py).
```
Comment (jemrobinson) [inline backend/README.md]: You had a better option for this in a previous revision: `createuser` or similar? Let's use that instead here.

File: backend/README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ You can test that `PostgreSQL` is correctly configured by running
 psql counterfactualcovid -U django -c "SELECT * FROM pg_user;"
 ```
 
-and checking that you get a list of users back.
+and checking that you get a list of users back.
```
Comment (jemrobinson) [inline backend/README.md]:
```suggestion
and checking that you get a list of users back.
```

**Review by jemrobinson — APPROVED** [review]

File: backend/README.md [hunk 3]
```diff
@@ -28,10 +28,17 @@ psql counterfactualcovid -c "CREATE USER django WITH PASSWORD 'value';" # where
...
 ```bash
-psql counterfactualcovid -c "SELECT * FROM pg_user;"
+psql counterfactualcovid -U django -c "SELECT * FROM pg_user;"
 ```
 
-and checking that you get a list of users back.
+and checking that you get a list of users back. If you get an error here is because the user `django` doesn't exist. To solve this just create a new user with:
+
+```bash
+createuser django -P
```
Comment (crangelsmith) [inline backend/README.md]: You mean this line @jemrobinson?

Comment (jemrobinson) [reply backend/README.md]: Yes. I think that's nicer than having to run a command inside `psql` plus also keeps the password out of the command line history.

File: backend/README.md (hunk 1 above)
Comment (crangelsmith) [reply backend/README.md]: ok implemented in 7d391d5

**Pull request merged**